en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -3,43 +3,41 @@
 Choreo’s API management capabilities allow you to create, publish, and manage all aspects of an API and its lifecycle.
 This quick start guide walks you through the steps to quickly create and publish an API using Choreo. 
 
-## Prerequisites:
+## Prerequisites
 - A mock REST service to use when creating the API. 
     
     !!!note
-        In this quick start guide, let's use the mock service [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed) that is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You can see the following JSON message:`{"hello": "world"}`
+        In this quick start guide, let's use the mock service at [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed), which is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You'll see the following JSON message: `{"hello": "world"}`.
  
 ## Step 1: Create an API from scratch
-Follow the procedure below to create an API using the mock REST service you tested in the prerequisites section:
+Follow this procedure to create an API using the mock REST service you tested in the prerequisites section:
 
-1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
-2. Sign in using either your GitHub or Google credentials.
-3. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
```

Comment:
Shall we consider re-phrasing this to "Sign in to the Choreo Console at via https://console.choreo.dev/"?

Reply:
+1

Reply:
Having a link at the end of a sentence is recommended to make the documentation more usable (since it is easier and quicker to click a link to the right).

Reply:
Applied suggestion in https://github.com/wso2/docs-choreo-dev/pull/108